src/pudl/analysis/service_territory.py

```diff
@@ -28,7 +28,7 @@
 MAP_CRS = "EPSG:3857"  # For mapping w/ OSM baselayer tiles
 CALC_CRS = "ESRI:102003"  # For accurate area calculations
 
-ENTITY_TYPE = {"ba": "balancing_authority", "util": "utility"}
+ENTITY_TYPE = {"ba": "balancing_authorities", "util": "utilities"}
```

Comment:
Tables that describe entities should be plural ("generators", "plants"...).

Reply:
Agreed